app/locales/ru.json

```diff
@@ -1,193 +1,193 @@
 {
   "main": {
     "toolTipHeader": "Stretchly - напоминания о перерывах",
-    "microbreakIn": "Mini Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "breakIn": "Long Break in $t(utils.seconds, {'count': {{seconds}} })",
-    "resumingBreaks": "Работа возобновлена",
+    "microbreakIn": "Мини-перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "breakIn": "Длинный перерыв через $t(utils.seconds, {'count': {{seconds}} })",
+    "resumingBreaks": "Продолжаем перерывы",
     "downloadLatestVersion": "Загрузить последнюю версию",
-    "toBreak": "Long Break",
-    "toMicrobreak": "Mini Break",
+    "toBreak": "длинному перерыву",
+    "toMicrobreak": "мини-перерыву",
     "skipToTheNext": "Перейти к ближайшему",
     "resume": "Возобновить",
     "pause": "Пауза",
-    "forHour": "час",
-    "for2Hours": "2 часа",
-    "for5Hours": "5 часов",
+    "forHour": "на час",
+    "for2Hours": "на 2 часа",
+    "for5Hours": "на 5 часов",
     "untilMorning": "до утра",
     "indefinitely": "на неопределённый срок",
     "resetBreaks": "Сбросить перерывы",
     "quitStretchly": "Выход из Stretchly",
-    "restoreDefaults": "Restore defaults",
-    "warning": "This will restore app defaults and all your settings will be lost.",
-    "continue": "Continue",
-    "cancel": "Cancel",
+    "restoreDefaults": "Восстановить настройки по умолчанию",
+    "warning": "Это действие вернет настройки к первоначальным, ваши настройки будут утеряны.",
+    "continue": "Продолжить",
+    "cancel": "Отменить",
     "preferences": "Настройки"
   },
   "break": {
-    "postpone": "Postpone this break",
-    "skip": "Skip this break",
+    "postpone": "Отложить",
+    "skip": "Пропустить",
     "title": "Перерыв!"
   },
   "preferences": {
-    "title": "Stretchly Preferences",
+    "title": "Настройки Stretchly",
     "nav": {
-      "settings": "Settings",
-      "schedule": "Schedule",
-      "theme": "Theme",
-      "about": "About",
+      "settings": "Настройки",
+      "schedule": "Расписание",
+      "theme": "Тема",
+      "about": "О программе",
       "heart": "Love Stretchly"
     },
     "settings": {
-      "openAtLogin": "Start Stretchly automatically when loggin in",
-      "showBreaksIn": "Shows breaks in:",
-      "window": "Window",
-      "fullscreen": "Full screen",
-      "showIdeas": "Show exercise tips during breaks",
-      "allScreens": "Shows breaks on all monitors",
-      "monitorIdleTime": "Monitor system idle time (countdown is paused if system is idle)",
-      "monitorDnd": "Show breaks even in Do Not Disturb mode",
-      "language": "Select langauge:",
-      "restoreDefaults": "Restore defaults"
+      "openAtLogin": "Запускать Stretchly автоматически при входе в систему",
+      "showBreaksIn": "Показывать перерывы:",
+      "window": "в окне",
```

Comment:
Lowercased and assuming context of `showBreaksIn`

Reply:
![image](https://user-images.githubusercontent.com/4745859/83414032-16b21d80-a41d-11ea-9829-272c7e774cb5.png)

Reply:
I guess if it's lowercase depends on Grammar of your langauge